src/main/java/ru/ayaz/User.java

```diff
@@ -0,0 +1,63 @@
+package ru.ayaz;
+
+import java.util.Set;
+import java.util.TreeSet;
+
+public class User {
+    private String nickname;
+    private Set<String> ignoredUsersSet;
+
+
+    public User() {
+        this.ignoredUsersSet = new TreeSet<String>();
```

Comment:
&#x1F34E; **Issue**
Why do you need a tree set ? Use hash set instead - it is a preferred default with the lover operations costs (O(1) compared to O(log(n))).